src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Legacy/Runtime.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    [Obsolete]
+    public static class Runtime
+    {
+        /// <summary>
+        /// Execute the provided string in the JavaScript context
+        /// </summary>
+        /// <returns>The js.</returns>
```

Comment:
The returns and param here are absolute nonsense, just delete them and leave the summary